ketcher-autotests/tests/utils/canvas/tools/helpers.ts

```diff
@@ -47,6 +47,30 @@ export async function selectSingleBondTool(page: Page) {
   await bondToolButton.click();
 }
 
+export async function selectEraseTool(page: Page) {
+  const bondToolButton = page.locator(`button[title*="Erase"]`);
+  await bondToolButton.click();
+}
+
+export async function selectRectangleSelectionTool(page: Page) {
+  const bondToolButton = page.locator(`button[title*="select-rectangle"]`);
```

Comment:
My proposal is to avoid using locators by title and use `page.getByTestId`

Reply:
All tools buttons are done in that way. Let's keep it in one style for now.